### ISSUE TITLE
Substitute models from Tensorflow or other platforms

### ISSUE THREAD

**Issue by qlin1** [davisking/dlib#898]
Is that possible to train my own Resnet face cognition model on Tensorflow, then transfer it into a dat file and just substitute the current "dlib_face_recognition_resnet_model_v1.dat" model?

Is that possible to let the constructor load the new model instead of the pretrained model then?

If not, is there any ways I can adjust the parameters or improve the accuracy of the pretrained face recognition model?

**Comment by xsacha** [comment]
See: #469

You don't have to use the pre-trained models. You can use dlib to train too.
